app/controllers/ToDo/PatchController.js

```diff
@@ -32,6 +33,10 @@ class PatchController {
             params: { id: todoId }
         } = request;
 
+        if (!(await this.isValidUuidServices.handle(todoId))) {
+            return response.sendStatus(StatusCodes.NOT_FOUND);
```

Comment:
It's not NOT_FOUND, it's BAD_REQUEST error!!!!!

Reply:
@rafal-ksiazek-rmtcfm-com  are you sure if there is 400? if user wants to test API and sends a request to `/users/something` then he receives 400 for any reason with a message that `UUID is not valid`, most of the users do not know what UUID is